src/redux/configureStore.ts

```diff
@@ -0,0 +1,25 @@
+import { applyMiddleware, createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
```

Comment:
fun fact: `redux-thunk` is super tiny https://github.com/gaearon/redux-thunk/blob/master/src/index.js